modules/nodes-descheduler-uninstalling.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * nodes/scheduling/nodes-descheduler.adoc
+// * nodes/scheduling/descheduler/nodes-descheduler-uninstalling.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nodes-descheduler-uninstalling_{context}"]
 = Uninstalling the descheduler
 
-You can remove the descheduler from your cluster by removing the descheduler instance and uninstalling the Kube Descheduler Operator. This procedure also cleans up the `KubeDescheduler` CRD and `openshift-kube-descheduler-operator` namespace.
+You can remove the descheduler from your cluster by removing the descheduler instance and uninstalling the {descheduler-operator}. This procedure also cleans up the `KubeDescheduler` CRD and `openshift-kube-descheduler-operator` namespace.
```

Comment:
@bergerhoffer Do you mean automatic clean up of `KubeDescheduler` CRD and `openshift-kube-descheduler-operator` namespace here when operator gets uninstalled ?

Reply:
@kasturinarra No - this means that in the procedure for uninstalling the operator below, we also have steps to delete the namespace and CRD. We've had this statement since we originally wrote the docs. This update is just replacing the operator name with an attribute

Reply:
@bergerhoffer okay, got it.. Thanks !!